### PR TITLE
jroot-patch-001

### PR DESCRIPTION
**Pull request by joshuaroot**
adding ‘custodian_snapshot’ tag to snapshot creation

### CODE REVIEW

**Comment by kapilt** [comment]
this branch needs updating

**Comment by joshuaroot** [comment]
Branch updated

File: c7n/resources/ec2.py [hunk 1]
```diff
@@ -475,7 +475,8 @@ def process_volume_set(self, resource):
             tags = [
                 {'Key': 'Name', 'Value': volume_id},
                 {'Key': 'InstanceId', 'Value': resource['InstanceId']},
-                {'Key': 'DeviceName', 'Value': block_device['DeviceName']}
+                {'Key': 'DeviceName', 'Value': block_device['DeviceName']},
+                {'Key': 'custodian_snapshot', 'Value': ['true']}
```
Comment (kapilt) [inline c7n/resources/ec2.py]: i would prefer a different value here, 'true' would need to be quoted/escaped in yaml as its a keyword for boolean which just makes matching more confusing. ie. think about what the policy looks like to match against these. Also you've made this a list when its a scalar value.

**Comment by kapilt** [comment]
lgtm

**Pull request merged**